From 18bf57c70aafcad031c0b43756b754dcaf6a756a Mon Sep 17 00:00:00 2001
From: Sebastian Freundt <hroptatyr@fresse.org>
Date: Sun, 07 Apr 2013 00:02:58 +0000
Subject: build-chain, replace INCLUDES directive by AM_CPPFLAGS

As of automake-13.1 the INCLUDES directive is no longer supported.
An automake run will return with an error.

This changeset simply follows automake's advice to replace INCLUDES
by AM_CPPFLAGS.
---
diff --git a/Tools.mk b/Tools.mk
index 56766da..f0fa0ec 100644
--- a/Tools.mk
+++ b/Tools.mk
@@ -32,7 +32,7 @@ TOOL=./$(DIR)$(EXEEXT_FOR_BUILD)
 
 EXTRA_DIST = $(TARG) $(TMPL) $(TSRC) $(DIST)
 
-INCLUDES = \
+AM_CPPFLAGS = \
 	   -I$(builddir) \
 	   -I$(srcdir) \
 	   -I$(top_builddir)/src \
@@ -43,7 +43,7 @@ INCLUDES = \
 	   $(WARN_CFLAGS)
 
 $(TOOL): $(TSRC) $(ALIAS_FILES)
-	$(AM_V_GEN) $(CC_FOR_BUILD) -o $(TOOL) $< $(INCLUDES)
+	$(AM_V_GEN) $(CC_FOR_BUILD) -o $(TOOL) $< $(AM_CPPFLAGS)
 
 $(TARG): $(TMPL) $(TSRC) $(DEPS)
 	$(AM_V_GEN) $(MAKE) $(TOOL) && \
diff --git a/fc-cache/Makefile.am b/fc-cache/Makefile.am
index 7f7d1b1..0e10786 100644
--- a/fc-cache/Makefile.am
+++ b/fc-cache/Makefile.am
@@ -36,7 +36,7 @@ uninstall-local:
 	-$(RM) -rf "$(DESTDIR)$(fc_cachedir)"
 endif
 
-INCLUDES=-I${top_srcdir} -I${top_srcdir}/src $(WARN_CFLAGS)
+AM_CPPFLAGS=-I${top_srcdir} -I${top_srcdir}/src $(WARN_CFLAGS)
 
 bin_PROGRAMS=fc-cache
 
diff --git a/fc-cat/Makefile.am b/fc-cat/Makefile.am
index b426723..04c1cc4 100644
--- a/fc-cat/Makefile.am
+++ b/fc-cat/Makefile.am
@@ -27,7 +27,7 @@ FC_CAT_SRC=${top_srcdir}/fc-cat
 
 SGML = ${FC_CAT_SRC}/fc-cat.sgml
 
-INCLUDES=-I${top_srcdir} $(WARN_CFLAGS)
+AM_CPPFLAGS=-I${top_srcdir} $(WARN_CFLAGS)
 
 bin_PROGRAMS=fc-cat
 
diff --git a/fc-list/Makefile.am b/fc-list/Makefile.am
index b2c499d..c58540e 100644
--- a/fc-list/Makefile.am
+++ b/fc-list/Makefile.am
@@ -29,7 +29,7 @@ SGML = ${FC_LIST_SRC}/fc-list.sgml
 
 bin_PROGRAMS=fc-list
 
-INCLUDES=-I${top_srcdir} $(WARN_CFLAGS)
+AM_CPPFLAGS=-I${top_srcdir} $(WARN_CFLAGS)
 
 BUILT_MANS=fc-list.1
 
diff --git a/fc-match/Makefile.am b/fc-match/Makefile.am
index 0e9e8fd..84afb8b 100644
--- a/fc-match/Makefile.am
+++ b/fc-match/Makefile.am
@@ -29,7 +29,7 @@ FC_MATCH_SRC=${top_srcdir}/fc-match
 
 SGML = ${FC_MATCH_SRC}/fc-match.sgml
 
-INCLUDES=-I${top_srcdir} $(WARN_CFLAGS)
+AM_CPPFLAGS=-I${top_srcdir} $(WARN_CFLAGS)
 
 BUILT_MANS=fc-match.1
 
diff --git a/fc-pattern/Makefile.am b/fc-pattern/Makefile.am
index 92b0433..c456247 100644
--- a/fc-pattern/Makefile.am
+++ b/fc-pattern/Makefile.am
@@ -29,7 +29,7 @@ FC_PATTERN_SRC=${top_srcdir}/fc-pattern
 
 SGML = ${FC_PATTERN_SRC}/fc-pattern.sgml
 
-INCLUDES=-I${top_srcdir} $(WARN_CFLAGS)
+AM_CPPFLAGS=-I${top_srcdir} $(WARN_CFLAGS)
 
 BUILT_MANS=fc-pattern.1
 
diff --git a/fc-query/Makefile.am b/fc-query/Makefile.am
index b3ea1e6..73b3f11 100644
--- a/fc-query/Makefile.am
+++ b/fc-query/Makefile.am
@@ -29,7 +29,7 @@ FC_QUERY_SRC=${top_srcdir}/fc-query
 
 SGML = ${FC_QUERY_SRC}/fc-query.sgml
 
-INCLUDES=-I${top_srcdir} $(FREETYPE_CFLAGS) $(WARN_CFLAGS)
+AM_CPPFLAGS=-I${top_srcdir} $(FREETYPE_CFLAGS) $(WARN_CFLAGS)
 
 BUILT_MANS=fc-query.1
 
diff --git a/fc-scan/Makefile.am b/fc-scan/Makefile.am
index 2063405..471a42f 100644
--- a/fc-scan/Makefile.am
+++ b/fc-scan/Makefile.am
@@ -29,7 +29,7 @@ FC_SCAN_SRC=${top_srcdir}/fc-scan
 
 SGML = ${FC_SCAN_SRC}/fc-scan.sgml
 
-INCLUDES=-I${top_srcdir} $(FREETYPE_CFLAGS) $(WARN_CFLAGS)
+AM_CPPFLAGS=-I${top_srcdir} $(FREETYPE_CFLAGS) $(WARN_CFLAGS)
 
 BUILT_MANS=fc-scan.1
 
diff --git a/fc-validate/Makefile.am b/fc-validate/Makefile.am
index 54edec2..782cead 100644
--- a/fc-validate/Makefile.am
+++ b/fc-validate/Makefile.am
@@ -29,7 +29,7 @@ FC_VALIDATE_SRC=${top_srcdir}/fc-validate
 
 SGML = ${FC_VALIDATE_SRC}/fc-validate.sgml
 
-INCLUDES=-I${top_srcdir} $(FREETYPE_CFLAGS) $(WARN_CFLAGS)
+AM_CPPFLAGS=-I${top_srcdir} $(FREETYPE_CFLAGS) $(WARN_CFLAGS)
 
 BUILT_MANS=fc-validate.1
 
diff --git a/src/Makefile.am b/src/Makefile.am
index 9fd7dd8..066cc03 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -70,7 +70,7 @@ uninstall-ms-import-lib:
 
 endif
 
-INCLUDES = 						\
+AM_CPPFLAGS = 						\
 	-I$(top_srcdir)					\
 	-I$(top_srcdir)/src				\
 	$(FREETYPE_CFLAGS)				\
--
cgit v0.9.0.2-2-gbebe
